src/components/atoms/button/RequestLeaveModalButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { DefaultButton } from '@components/atoms';
+import { useModalStore } from '@stores/layerStore';
+import { useNavigate } from 'react-router-dom';
+
+const RequestLeaveModalButton = () => {
+	const { openModal } = useModalStore();
+	const navigate = useNavigate();
+
+	const goBack = () => {
+		navigate(-1);
+	};
+	const stay = () => {};
+
+	const buttons = [
+		<DefaultButton
+			color={{
+				textColor: 'White',
+				bgColor: 'Primary',
+			}}
+			title="머무르기"
+			size="small"
+			onClick={stay}
+		/>,
+		<DefaultButton
+			title="이동하기"
+			size="small"
+			color={{
+				textColor: 'Primary',
+				bgColor: 'White',
+				borderColor: 'Primary',
+			}}
+			onClick={goBack}
+		/>,
+	];
+
+	const handleClickRequestLeaveModalButton = (
+		e: React.MouseEvent<HTMLDivElement, MouseEvent>,
```

Comment:
이전에 합의했던 대로 React를 import 구문으로 빼내어 사용하려고 하였으나 오류가 나는 문제가 생겼습니다. 이에 `React`를 앞에 명시하게 되었는데 혹시 적용할 수 있는 방법 말씀해주신다면 변경해보도록 하겠습니다!

Reply:
import { MouseEvent } from 'react';
```suggestion
		e: MouseEvent<HTMLDivElement>,
```

이렇게하면 동일해지는 것 같습니다!

Reply:
오 업데이트하도록 하겠습니다!